examples/display_button_customfont.py

```diff
@@ -11,6 +11,8 @@
 import adafruit_touchscreen
 from adafruit_button import Button
 
+display = board.DISPLAY
```

Comment:
Can you include this comment about the display setup here please
```
# use built in display (PyPortal, PyGamer, PyBadge, CLUE, etc.)
# see guide for setting up external displays (TFT / OLED breakouts, RGB matrices, etc.)
# https://learn.adafruit.com/circuitpython-display-support-using-displayio/display-and-display-bus
display = board.DISPLAY
```
This helps folks discover the relevant guide page for setting up external displays if that is what they are using.
```

Reply:
@FoamyGuy Thanks, I added the note in the examples, and add the MagTag to the list also

Reply:
https://github.com/adafruit/Adafruit_CircuitPython_Display_Button/pull/30/commits/dd4fea43a45dbf355bf12b84d2ff35715d9cfe0d